server/spec/gitlab_request_auth_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'rack'
+
+include TxghServer
+
+describe GitlabRequestAuth do
+  let(:secret) { 'abc123' }
+
+  describe '.authentic_request?' do
+    it 'returns true if the request is signed correctly' do
```

Comment:
It looks like Gitlab auth works via shared secret. Can we update this description to reflect that instead of saying the request is "signed"?